cluster/libvirt-coreos/skydns-rc.yaml

```diff
@@ -1,44 +1,61 @@
+apiVersion: v1beta3
 kind: ReplicationController
-apiVersion: v1beta1
-id: skydns
-namespace: default
-labels:
-  k8s-app: skydns
-desiredState:
+metadata:
+  labels:
+    k8s-app: skydns
+  name: skydns
+  namespace: default
+spec:
   replicas: ${DNS_REPLICAS}
-  replicaSelector:
+  selector:
     k8s-app: skydns
-  podTemplate:
-    labels:
-      k8s-app: skydns
-    desiredState:
-      manifest:
-        version: v1beta2
-        id: skydns
-        dnsPolicy: "Default"  # Don't use cluster DNS.
-        containers:
-          - name: etcd
-            image: quay.io/coreos/etcd:latest
-            command: [
-                    \"/etcd\",
-                    \"-bind-addr=127.0.0.1\",
-                    \"-peer-bind-addr=127.0.0.1\",
-            ]
-          - name: kube2sky
-            image: kubernetes/kube2sky:1.0
-            command: [
-                    # entrypoint = \"/kube2sky\",
-                    \"-domain=${DNS_DOMAIN}\",
-            ]
-          - name: skydns
-            image: kubernetes/skydns:2014-12-23-001
-            command: [
-                    # entrypoint = \"/skydns\",
-                    \"-machines=http://localhost:4001\",
-                    \"-addr=0.0.0.0:53\",
-                    \"-domain=${DNS_DOMAIN}.\",
-            ]
-            ports:
-              - name: dns
-                containerPort: 53
-                protocol: UDP
+  template:
+    metadata:
+      labels:
+        k8s-app: skydns
+    spec:
+      containers:
+      - args:
+        - \"/etcd\"
+        - \"-bind-addr=127.0.0.1\"
+        - \"-peer-bind-addr=127.0.0.1\"
+        capabilities: {}
```

Comment:
Please remove fields containing default values. The conversion script doesn't do that currently. If the field wasn't specified previously, it probably doesn't need to be specified in v1beta3, either, especially if it's just empty/false/0.